### ISSUE TITLE
An end2end demo will be helpful.

### ISSUE THREAD

**Issue by simonxue** [v-v/BiDNN#1]
imageset + textset, train, test, pretrained models, etc. :-).

**Comment by v-v** [comment]
Hi Simon,

Thank you for your comment - I definitely agree. The problem is that TRECVID's (and before them, Mediaeval's) datasets are very closed, restrictive and tricky (BBC didn't even let them be used for TRECVID this year).

Around this spring, I plan to test the method with Flickr8k or Flickr30k and similar datasets. It will be interesting to see how well it works outside video hyperlinking (where it seems to be working quite well, given this year TRECVID's live evaluation on AMT) and it will allow me to also make a complete setup that people can run, test, retrain and do everything easier. End-to-end training (as well as trying a few classic tricks like batch normalization, testing sparsity etc.) is also something I plan to do as part of the work I'll do this spring :)

I'll update this issue on github when I do.

Kind regards,
Vedran

**Comment by simonxue** [comment]
I look forward to seeing the testing result. Thanks.

2016-12-26 19:49 GMT+08:00 Vedran Vukotic <notifications@github.com>:

**Comment by Supersak80** [comment]
Hi @v-v , any updates on being able to release a demo with sample train/test inputs for two modalities?

**Comment by ngbrenda** [comment]
Yes, a demo with sample data to illustrate training would be super useful. Any updates?
